From 83752d01443ee6a0a046958b450b841449fc2edc Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Thu, 29 Jun 2017 15:26:41 +0300
Subject: [PATCH 1055/1345] fix: comphy: fix incorrect defines values

commit  d49fc2ac71c8f508216d1aaf7196fa467810ee99 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

fix incorrect COMMON_PHY_CFG1_CORE_RSTN_OFFSET value, set to 13.
fix incorrect COMMON_PHY_CFG1_PWR_ON_RESET_OFFSET value, set to 14.

Change-Id: I488e4b23d1ec1f6e2a2b4c15c7aa8aa48445305b
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40992
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/phy/phy-comphy-cp110.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/phy/phy-comphy-cp110.h b/drivers/phy/phy-comphy-cp110.h
index a836a25..c523bef 100644
--- a/drivers/phy/phy-comphy-cp110.h
+++ b/drivers/phy/phy-comphy-cp110.h
@@ -13,10 +13,10 @@
 #define COMMON_PHY_CFG1_PWR_UP_MASK		(0x1 << COMMON_PHY_CFG1_PWR_UP_OFFSET)
 #define COMMON_PHY_CFG1_PIPE_SELECT_OFFSET	2
 #define COMMON_PHY_CFG1_PIPE_SELECT_MASK	(0x1 << COMMON_PHY_CFG1_PIPE_SELECT_OFFSET)
-#define COMMON_PHY_CFG1_PWR_ON_RESET_OFFSET	13
-#define COMMON_PHY_CFG1_PWR_ON_RESET_MASK	(0x1 << COMMON_PHY_CFG1_PWR_ON_RESET_OFFSET)
-#define COMMON_PHY_CFG1_CORE_RSTN_OFFSET	14
+#define COMMON_PHY_CFG1_CORE_RSTN_OFFSET	13
 #define COMMON_PHY_CFG1_CORE_RSTN_MASK		(0x1 << COMMON_PHY_CFG1_CORE_RSTN_OFFSET)
+#define COMMON_PHY_CFG1_PWR_ON_RESET_OFFSET	14
+#define COMMON_PHY_CFG1_PWR_ON_RESET_MASK	(0x1 << COMMON_PHY_CFG1_PWR_ON_RESET_OFFSET)
 
 #define COMMON_PHY_CFG6_REG			0x14
 #define COMMON_PHY_CFG6_IF_40_SEL_OFFSET	18
-- 
1.7.9.5

